test/unit/test_compression_util.py

```diff
@@ -2,20 +2,33 @@
 import tempfile
 import unittest
 
-from galaxy.util.compression_utils import CompressedFile
+from galaxy.util.compression_utils import CompressedFile, get_fileobj_raw
```

Comment:
```suggestion
import six

from galaxy.util.compression_utils import (
    CompressedFile, 
    get_fileobj_raw,
)
```

This is just our import convention, and six is needed to assert we get the correct type from  `get_fileobj_raw `